### PR TITLE
Added Central CLI overview and how to register an API

### PR DESCRIPTION
**Pull request by DesertLeprechaun**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by DesertLeprechaun** [comment]
Added CLI overview

File: content/en/docs/integrate_with_central/cli_central/cli_register_api.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+* [curl](https://curl.se/) is installed on the system
+* [jq](https://stedolan.github.io/jq/) is installed on the system
+
+## Objectives
+
+Register an API in Amplify Central using the Axway Central CLI.
+
+* Create a new API service in an environment
+* Register an API to the service
+* Publish API to the Unified Catalog
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/cli_central/cli_register_api.md:27]: We should probably say "Publish an API to the Asset Catalog, Product Foundry and Marketplace (formerly the Unified Catalog)."

File: content/en/docs/integrate_with_central/cli_central/cli_register_api.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+```bash
+#!/bin/bash
+export serviceName=`jq -r .[0].name api-service-created.json`
+export envName=`jq -r .[0].metadata.scope.name api-service-created.json`
+export encodedImage=`base64 -i api.png`
+axway central get apis $serviceName --scope $envName -o json | jq '.spec.icon.data = $ENV.encodedImage' | jq '.spec.icon.contentType = "image/png"'  > upload.json
+axway central apply -f upload.json
+```
+
+### Publish the API to the Unified Catalog
```
Comment (robertengaxway) [inline content/en/docs/integrate_with_central/cli_central/cli_register_api.md:191]: I would suggest another section for "Publish the API to the Asset Catalog".

**Review by robertengaxway — APPROVED** [review]
I approve

File: content/en/docs/integrate_with_central/cli_central/cli_register_api.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/integrate_with_central/cli_central/cli_register_api.md:186]: This was covered by the Create an Asset in another section

**Review by robertengaxway — APPROVED** [review]
I approve

**Pull request merged**